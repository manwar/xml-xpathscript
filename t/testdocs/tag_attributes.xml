<doc>
<foo bar="baz">
    <a/>
    <b/>
</foo>
</doc>
