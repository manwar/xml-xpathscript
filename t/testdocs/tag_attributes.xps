<%

$template->set( 'foo' => {
                   ( map { $_ => "$_\n" } qw/ pre post intro extro 
                                            prechild postchild
                                            prechildren postchildren / ),
                   showtag => 1,
                   action => DO_SELF_ONLY,
                   testcode => sub { DO_SELF_AND_KIDS },
                 });

$template->set( 'text()' => { showtag => 0 } );
%>
<%= apply_templates() %>
