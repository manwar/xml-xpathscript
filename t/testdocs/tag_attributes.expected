
<doc>pre
<foo bar="baz">intro
prechildren
prechild
<a></a>postchild
prechild
<b></b>postchild
postchildren
extro
</foo>post
</doc>
